File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.cs
using System;

class Program {
    static void Main() {
        string[] inputs = Console.ReadLine().Split();
        int N = int.Parse(inputs[0]);
        int M = int.Parse(inputs[1]);
        int[] arr = new int[N];
        
        for (int x = 0; x < N; x++)
            arr[x] = x + 1;

        for (int y = 0; y < M; y++) {
            string[] swapInputs = Console.ReadLine().Split();
            int i = int.Parse(swapInputs[0]);
            int j = int.Parse(swapInputs[1]);

            int temp = arr[i - 1];
            arr[i - 1] = arr[j - 1];
            arr[j - 1] = temp;
        }

        Console.WriteLine(string.Join(" ", arr));
    }
}

File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.js
const input = require('fs').readFileSync('/dev/stdin').toString().split('\n');
const [N, M] = input[0].split(' ').map(Number);
let arr = Array.from({ length: N }, (_, index) => index + 1);

for (let i = 1; i <= M; i++) {
    const [x, y] = input[i].split(' ').map(Number);
    let temp = arr[x - 1];
    arr[x - 1] = arr[y - 1];
    arr[y - 1] = temp;
}

console.log(arr.join(' '));